File: Elmohandes-Service-Center/front-end/src/model/final-bill-vm.ts
import {Customer} from "./customer";
import {Car} from "./car";
import {Bill} from "./bill";
import {BillFields} from "./bill-fields";

export interface FinalBillVm {
  customerDto: Customer;
  carDto: Car;
  billDto: Bill;
  billFieldsDtos: BillFields[];
}

File: Elmohandes-Service-Center/back-end/ElmohandesServiceCenter/src/main/java/com/example/elmohandesservicecenter/controller/vm/FinalBillVm.java
package com.example.elmohandesservicecenter.controller.vm;

import com.example.elmohandesservicecenter.dto.BillDto;
import com.example.elmohandesservicecenter.dto.BillFieldsDto;
import com.example.elmohandesservicecenter.dto.CarDto;
import com.example.elmohandesservicecenter.dto.CustomerDto;
import com.example.elmohandesservicecenter.dto.jwt.ManagerDto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class FinalBillVm {
    private CustomerDto customerDto;
    private CarDto carDto;
    private BillDto billDto;
    private List<BillFieldsDto> billFieldsDtos;
}
